common/src/main/java/com/wynntils/commands/LootrunCommand.java

```diff
@@ -403,7 +403,7 @@ public LiteralArgumentBuilder<CommandSourceStack> getBaseCommandBuilder() {
 
     private int screenLootrun(CommandContext<CommandSourceStack> context) {
         // Delay is needed to prevent chat screen overwriting the lootrun screen
-        Delay.create(() -> McUtils.mc().setScreen(WynntilsLootrunsScreen.create()), 1);
+        Managers.TickScheduler.scheduleLater(() -> McUtils.mc().setScreen(WynntilsLootrunsScreen.create()), 2);
```

Comment:
This could have been 1 tick, I am pretty sure. Doesn't matter though

Reply:
I think so to, but out of an abundance of caution I did not want to change the actual behavior in a patch that was hastily merged in the middle of the night. :-) You are welcome to fix it though.